changelog.txt

```diff
@@ -1,3 +1,11 @@
+4.11.0
+============
+Feature Requests: **MSAL.NET will now remove accounts from the cache that have expired refresh tokens** MSAL.NET will remove the refresh token and the associated account if the if suberror is "bad_token" to avoid overloading ESTS. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1720)
```

Comment:
**MSAL.NET will now remove accounts from the cache that have expired refresh tokens**. MSAL.NET will remove both the refresh token and the associated account if the if `suberror` is "bad_token" to unnecessary calls to AzureAD. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1720)